src/main.rs

```diff
@@ -0,0 +1,65 @@
+mod config;
+mod client;
+
+use std::{env, process};
+
+fn main() {
```

Comment:
More practically we can have `main` return an error type with `fn main() -> Result<(), Box<dyn std::error::Error>>` or use something like [`eyre`](https://crates.io/crates/eyre) to propagate a stack of errors in a more readable format.